selfdrive/sensord/sensors_qcom2.cc

```diff
@@ -27,17 +28,22 @@
 
 ExitHandler do_exit;
 std::mutex pm_mutex;
-
-// filter first values (0.5sec) as those may contain inaccuracies
 uint64_t init_ts = 0;
-constexpr uint64_t init_delay = 500*1e6;
 
-void interrupt_loop(int fd, std::vector<Sensor *>& sensors, PubMaster& pm) {
+void send_message(PubMaster& pm, MessageBuilder& msg, std::string &service) {
+  std::lock_guard<std::mutex> lock(pm_mutex);
+  pm.send(service.c_str(), msg);
+}
```

Comment:
we should just use separate PubMasters for each loop

Reply:
done